src/utils/searchInDatabase.js

```diff
@@ -0,0 +1,33 @@
+import db from '../models';
+import ResponceHandler from './ResponseHandler';
+/**
+ *
+ *
+ * @export
+ * @class DatabaseSearch
+ */
+class DatabaseSearch {
+/**
+ *
+ *
+ * @static
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Object} next
```

Comment:
Lets have this as a User controller, instead of a util.

Reply:
sure. Am going to  update it.